pcsx2/USB/usb-pad/usb-buzz.cpp

```diff
@@ -198,6 +198,42 @@ namespace usb_pad
 		return "BuzzDevice";
 	}
 
+	bool BuzzDevice::Freeze(USBDevice* dev, StateWrapper& sw) const
+	{
+		BuzzState* s = USB_CONTAINER_OF(dev, BuzzState, dev);
+		if(!sw.DoMarker("BuzzDevice"))
+			return false;
+
+		sw.Do(&s->data.head1);
```

Comment:
head1/2 are constant and the button presses are usually not restored.

Reply:
Cool. Better that way as we don't have to bump the savestate version :+1: